pkg/planner/cardinality/row_count_index.go

```diff
@@ -170,15 +170,15 @@ func getIndexRowCountForStatsV1(sctx context.PlanContext, coll *statistics.HistC
 			}
 			var count float64
 			var err error
-			colIDs := coll.Idx2ColumnIDs[idxID]
+			colIDs := coll.Idx2ColUniqueIDs[idxID]
```

Comment:
same as above

Reply:
Index ID is not relevant here, only Column ID has this difference.